builder/dockerfile/builder.go

```diff
@@ -142,6 +142,9 @@ func sanitizeRepoAndTags(names []string) ([]reference.Named, error) {
 
 		if _, isTagged := ref.(reference.NamedTagged); !isTagged {
 			ref, err = reference.WithTag(ref, reference.DefaultTag)
+			if err != nil {
+				return nil, err
+			}
```

Comment:
wow.. I thought you were removing the err != nil check last night when I commented hence my horribly confusing comment.. temporarily forgot what green and "**+**" means!? :-/ 

I wonder if this was ignored because appending the "`:latest`" tag to a reference is probably highly unlikely to error out. But unchecked errors seem pretty non-Go-like.